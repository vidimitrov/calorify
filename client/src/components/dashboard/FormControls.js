import styled from 'styled-components';

const FormControls = styled.div`
  margin-bottom: 40px;
`;

export default FormControls;
