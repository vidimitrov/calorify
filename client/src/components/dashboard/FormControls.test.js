import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import FormControls from './FormControls';

it('Dashboard/FormControls component renders correctly', () => {
  const tree = renderer.create(<FormControls />).toJSON();
  expect(tree).toMatchSnapshot();
});
