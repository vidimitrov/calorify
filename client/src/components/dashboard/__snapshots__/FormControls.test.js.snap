// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard/FormControls component renders correctly 1`] = `
.c0 {
  margin-bottom: 40px;
}

<div
  className="c0"
/>
`;
